tests/FakeItEasy.Specs/ThrowsAsyncSpecs.cs

```diff
@@ -304,7 +304,6 @@ public static void WithResultPassExceptionFactoryWithFourArgs(IFoo fake, Task<in
                 .x(() => task.Exception?.InnerException.Should().BeAnExceptionOfType<MyException>());
         }
 
-        [SuppressMessage("Microsoft.Usage", "CA2237:MarkISerializableTypesWithSerializable", Justification = "Irrelevant for test purposes, and not compatible with .NET Core")]

```

Comment:
Any idea why we no longer need this warning?

Reply:
Great question. It's currently superfluous in `develop`.

My guess is that it's because the rule wasn't ported to the Roslyn-based FxCop: https://github.com/dotnet/roslyn-analyzers/issues/641.
If we make the other exceptions non-serializable, we'll get hit by the error all over the place once they port it, I expect. I can leave it if you prefer, or just address it if

1. we decide to make exceptions non-serializable, and
2. the rule is eventually ported and we pick up the new version of the analyzer